src/controllers/chat.controller.js

```diff
@@ -1,8 +1,10 @@
 const path = require('path');
-
 const db = require(path.join(__dirname, '../db/index'));
 const queries = require(path.join(__dirname, '../db/queries'));
 const getPayload = require(path.join(__dirname, '../helpers/jwt'));
+const request = require('request');
+
+const emojiApiUrl = 'http://127.0.0.1:5000/';
```

Comment:
will need to change this later

Reply:
No worries. We should be able to serve the flask API on the droplet just FYI.

Reply:
Not sure where to flag this, but don't forget to update the apidoc comments in routes/chats.js.

Reply:
cool i was wondering if we could serve both on the droplet 👍